docs/source/asr/configs.rst

```diff
@@ -218,6 +218,31 @@ For example, a decoder config corresponding to a sub-word tokenization model sho
       vocabulary: []  # filled with vocabulary from tokenizer at runtime
 
 
+Dynamic Layer Freezing
+---------------------------------
```

Comment:
The dashes should have the same length as the title.

Reply:
Fixed